modules/iam-bootstrap/bootstrap-2.json

```diff
@@ -0,0 +1,78 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "GlueUngated",
+      "Effect": "Allow",
+      "Action": [
+        "glue:CreateDatabase",
+        "glue:GetDatabase",
+        "glue:DeleteDatabase",
+        "glue:CreateSecurityConfiguration",
+        "glue:GetSecurityConfiguration",
+        "glue:DeleteSecurityConfiguration",
+        "glue:GetTags",
+        "glue:CreateCrawler",
+        "glue:CreateTable",
+        "glue:GetTable",
+        "glue:DeleteTable",
+        "glue:GetCrawler",
+        "glue:DeleteCrawler"
+      ],
+      "Resource": "*"
```

Comment:
can we gate resources to specific deploy id? Also keep in mind `iam_bootstrap`   is for deployment infra only.